configuration/contrib/collectd/tedge-collectd.conf

```diff
@@ -1654,3 +1654,30 @@ LoadPlugin mqtt
 <Include "/etc/collectd/collectd.conf.d">
 	Filter "*.conf"
 </Include>
+
```

Comment:
For testing, add this snippet to the **end** of your existing `collectd.conf` file